src/backend/CommandBufferStateTracker.cpp

```diff
@@ -542,10 +549,20 @@ namespace backend {
         return true;
     }
 
+    void CommandBufferStateTracker::ResetDependentPipelineApsects() {
```

Comment:
nit: `DependentPipeline`->`PipelineDependent` and `Apsects`->`Aspects` if this function is kept